commons/addons/fs_buckets.md

```diff
@@ -41,14 +41,28 @@ You will be able to retrieve generated data between two deployments.
 </tr>
 <tr>
 <td class="cc-col__price "><span class="label cc-label__price label-info">S</span></td>
-<td>UNLIMITED</td>
+<td>1 GB</td>
 <td>UNLIMITED</td>
 <td>1.50€</td>
 </tr>
 </table>
 
 ## Configuring your application
 
+Buckets are configured using environment variables. Add the following to your application :
+
+```
+CC_FS_BUCKET=/some/empty/folder:bucket-8cef666d-6c0f-406e-bd65-a12877a30fba-fsbucket.services.clever-cloud.com
```

Comment:
I see that this fs-bucket exists, maybe it would be better to use a uuid like the one you used bellow (012345678-....)

Reply:
Oh ! This is the example gived in admin area, it's my test bucket. Sorry.